WooCommerce/src/main/kotlin/com/woocommerce/android/ui/bookings/filter/datetime/DateTimeFilterPage.kt

```diff
@@ -112,8 +115,21 @@ private fun DateTimeSection(
     timeReadable: String,
     onDateClick: () -> Unit,
     onTimeClick: () -> Unit,
+    onClearClick: () -> Unit,
 ) {
-    BookingSectionHeader(header = header)
+    BookingSectionHeader(
+        header = header,
+        action = {
+            if (dateReadable.isNotEmpty()) {
+                Text(
+                    text = stringResource(R.string.clear).uppercase(),
```

Comment:
```suggestion
                    text = stringResource(R.string.clear).uppercase(),
                    text = stringResource(R.string.clear).uppercase(Locale.getDefault()),
```

Let's use Locale when making the text uppercase.

Reply:
Done